lms/urls.py

```diff
@@ -995,6 +995,12 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    if getattr(settings, "EDX_GRADES_STORAGE_MEDIA_URL", None) \
```

Comment:
Does it backslash? Please take a look at open edx python styleguide

Reply:
Done